template_feed/Microsoft.DotNet.Test.ProjectTemplates.8.0/content/NUnit-FSharp/Program.fs

```diff
@@ -1,4 +1,4 @@
-﻿module Program =
```

Comment:
Probably a nitpick, but if we remove = after module, then we should de-indent the rest of the file.